util/gpu.py

```diff
@@ -5,4 +5,4 @@ def get_available_gpus():
     Returns the number of GPUs available on this system.
     """
     local_device_protos = device_lib.list_local_devices()
-    return [x.name for x in local_device_protos if x.device_type == 'GPU']
+    return [x for x in local_device_protos if x.device_type == 'GPU']
```

Comment:
Is there any reason for that? `pydoc tensorflow.python.client` nor the official doc gives any hint on anything :(

Reply:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/device_attributes.proto

Reply:
This is for getting access to `memory_limit`.